pointcloud/src/main/scala/geotrellis/pointcloud/raster/ept/DEMRasterSource.scala

```diff
@@ -219,7 +217,7 @@ case class DEMRasterSource(
           }
         }
 
-        tris.asScala.foreach(rasterizeTriangle(_))
+        tris.asArray.foreach(rasterizeTriangle)
```

Comment:
`asScala` converts Native Iterator into Scala iterator. The conversion `.toList` / `foreach` will pull down data from the C memory on the each step. [asArray](https://github.com/PDAL/java/blob/master/core/src/main/scala/io/pdal/TriangularMesh.scala#L43) converts all the mesh into Java Array in a single copy.